XML/syntax_test_xml.xml

```diff
@@ -17,9 +17,17 @@
 <!--                     ^^ punctuation.definition.tag.end -->
 <!--                       ^ - meta.tag.preprocessor -->
 
-
-     <?XML version="1.0" ?>
-<!-- ^ invalid.illegal.missing-entity - meta.tag.preprocessor -->
+     <?XML VERSION="1.0" ?>
+<!-- ^^^^^^^^^^^^^^^^^^^^^^ meta.tag.preprocessor -->
```

Comment:
this is not valid in XML - try it on https://codebeautify.org/xmlvalidator or http://www.xmlvalidation.com/ or read the RFC. The behavior prior to this commit was correct.

> The processing instruction target matching "[xX][mM][lL]" is not allowed.